dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -803,8 +803,15 @@ public Workflow publish(@ApiParam(hidden = true) @Parameter(hidden = true, name
     public Workflow getWorkflowByPath(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User user,
             @Parameter(name = "repository", description = "Repository path", required = true, in = ParameterIn.PATH) @ApiParam(value = "repository path", required = true) @PathParam("repository") String path,
             @Parameter(name = "include", description = "Comma-delimited list of fields to include: " + VALIDATIONS + ", " + ALIASES, in = ParameterIn.QUERY) @ApiParam(value = "Comma-delimited list of fields to include: " + VALIDATIONS + ", " + ALIASES) @QueryParam("include") String include,
-            @Parameter(name = "subclass", description = "Which Workflow subclass to retrieve. One of the folowwing: " + SERVICE + ", " + BIOWORKFLOW +  ", " + APPTOOL, in = ParameterIn.QUERY, required = true) @ApiParam(value = "Which Workflow subclass to retrieve. One of the following: " + SERVICE + ", " + BIOWORKFLOW +  ", " + APPTOOL) @QueryParam("subclass") String subclass) {
-        final Class<? extends Workflow> targetClass = getSubClass(subclass);
+            @Parameter(name = "subclass", description = "Which Workflow subclass to retrieve. One of the following: " + SERVICE + ", " + BIOWORKFLOW +  ", " + APPTOOL, in = ParameterIn.QUERY, required = true) @ApiParam(value = "Which Workflow subclass to retrieve. One of the following: " + SERVICE + ", " + BIOWORKFLOW +  ", " + APPTOOL, required = true) @QueryParam("subclass") String subclass,
+            @Parameter(name = "services", description = "Should only be used by Dockstore CLI versions < 1.11.0. Indicates whether to get a service or workflow", in = ParameterIn.QUERY, hidden = true) @ApiParam(value = "services", hidden = true) @QueryParam("services") Boolean services) {
```

Comment:
Shouldn't this be less than 1.12.0? 1.11.x is still using the boolean param, isn't it?

Reply:
There is a `deprecated` field for OpenAPI you can use. Doesn't look like there is one for Swagger.